README.md

```diff
@@ -501,9 +513,9 @@ var handle = await client.StartWorkflowAsync(
 Console.WriteLine(
     "Current greeting: {0}",
     await handle.QueryWorkflowAsync(wf => wf.CurrentGreeting()));
-// Change the params via signal
-var signalArg = new GreetingParams(Salutation: "Aloha", Name: "John");
-await handle.SignalWorkflowAsync(wf => wf.UpdateGreetingParamsAsync(signalArg));
+// Change the params via update
+var updateArg = new GreetingParams(Salutation: "Aloha", Name: "John");
+await handle.ExecuteUpdateAsync(wf => wf.UpdateGreetingParamsAsync(updateArg));
```

Comment:
I don't think that's necessary in this README, I think that can happen when this is properly documented. Note I am not showing you can set an ID either.

Reply:
It's worth it IMHO, otherwise it's unclear how this is different from a signal. Of course it'll be properly documented later but for a Temporal noob this would be a bit confusing.

Reply:
Agreed. No reason _not_ to show it, and at least this reason _to_ show it.

Reply:
The reason is reworking temporary documentation for this part. I don't think here is where users should differentiate updates and signals, otherwise we have lots to add here. Why do I not have to show that queries accept parameters? Or that updates can have validators? Users may get confused reading this on how a query is different than an update by this same logic.

I will add it, but I think you're looking too closely at an intentionally limited sample in a temporary README.

Reply:
Hrmm, I'm struggling to fit the update-with-return value use case into the above sample. Any ideas on what I can change the `GreetingWorkflow`'s `UpdateGreetingParamsAsync` to for demonstrating a return value?